<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>885</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>885</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(56, 58, 89);	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>885</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>885</width>
     <height>575</height>
    </size>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;영상 다운로드&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;PDF 추출&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;OCR 적용&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;단어장&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="anyang">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	border-image: url(&quot;./ui/image/anyang.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: left;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="youtube">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	border-image: url(&quot;./ui/image/youtube.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	border-image: url(&quot;./ui/image/pdf.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ocr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image:url(&quot;./ui/image/ocr.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="flashcard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image:url(&quot;./ui/image/flash-cards.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="calendar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
border-image:url(&quot;./ui/image/calendar.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bbs">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{
border-image:url(&quot;./ui/image/chat.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="quiz">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{
border-image:url(&quot;./ui/image/speech-bubble.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;캘린더&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget{
	background-color: rgb(135, 135, 181);
	border-radius:30px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;왼쪽 아이콘을 클릭해 기능을 선택하세요 !!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>521</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	border-image: url(&quot;./ui/image/study.png&quot;);
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="aw">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="Home">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>홈페이지</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OK">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>다운로드</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;꼭 실행 시 켜진 홈페이지에서 로그인 후 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;영상을 실행해주시고 버튼을 눌러주세요.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="yw">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="URL_textBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;youtube 주소 입력&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OK_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;

}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>확인</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pw">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="OK_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>확인</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Video_textBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>ex) 캡스톤디자인.mp4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;다운로드 받을 영상이름 입력&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fw">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="DEF_textBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>ex)물</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;단어 입력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;단어 정의 입력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WORD_textBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;</string>
      </property>
      <property name="text">
       <string>ex)water</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OK_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>완료</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Create">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>생성</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fqw">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="Answer_textBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>390</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{

	font: 75 9pt &quot;CookieRun Bold&quot;;

}</string>
      </property>
      <property name="text">
       <string>ex) test</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>360</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;정답 입력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="Picture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	border-radius: 40px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;

}</string>
      </property>
      <property name="text">
       <string>사진 나오는 곳</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OK_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>380</y>
        <width>93</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;	
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>확인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Start">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-position: center;
    background-repeat: no-repeat;
	border: none;
	background-color: rgb(56, 58, 89);
	text-align: center;
	border-radius: 10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;
}
QPushButton:hover {
	background-color: rgb(33, 37, 43);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>문제 시작</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>285</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;단어 퀴즈&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;CookieRun Bold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;커뮤니티&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
